images/scripts/opensuse-tumbleweed.setup

```diff
@@ -21,11 +21,15 @@ printf 'dictcheck = 0\nminlen = 6\n' >> /etc/security/pwquality.conf
 COCKPIT_DEPS="\
 criu \
 libcriu2 \
+crypto-policies-scripts \
```

Comment:
Please add "... on openshift-tumbleweed" to the "fix missing dependencies" commit.